### PR TITLE
5.0.7-rt5

### PR DESCRIPTION
**Pull request by tex**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ sudo -i nix-channel --update musnix
 `musnix.kernel.packages`
 * **Description:** This option allows you to select the real-time kernel used by musnix.
 * **Type:** `package`
-* **Default value:** `pkgs.linuxPackages_4_9_rt`
+* **Default value:** `pkgs.linuxPackages_5_0_rt`
```
Comment (magnetophon) [inline README.md]: Why do you make 5.0 the default kernel?

File: modules/kernel.nix [hunk 2]
```diff
@@ -11,10 +11,13 @@ let
     SCHEDSTATS y
   '';
 
-  kernelConfigOptimize = ''
+  kernelConfigOptimizeDeadline = ''
```
Comment (magnetophon) [inline modules/kernel.nix]: Why this change?

File: README.md (hunk 1 above)
Comment (tex) [reply README.md]: Why not? Newer is always better. Better security better performance better features ;-)  (just kidding)

File: modules/kernel.nix (hunk 2 above)
Comment (tex) [reply modules/kernel.nix]: Because in 5.0+ there is no such option anymore. It is purely run time option now as far as I understand.

Comment (magnetophon) [reply modules/kernel.nix]: OK, thanks.

**Comment by tex** [comment]
Ping.

**Comment by henrytill** [comment]
Hey @tex, sorry for the latency.  

I will review this over the weekend.

**Comment by tex** [comment]
Ping.

**Comment by henrytill** [comment]
Sorry about the latency @tex.  Thanks for pinging again.
